README.md

```diff
@@ -1,88 +1,126 @@
+
+**Note: if your app uses Objective-C see [video-quickstart-objective-c](https://github.com/twilio/video-quickstart-objc/).**
 
 The sample apps are written in Swift 3.0. You will need at least Xcode 8.0 in order to run the applications.
 
 Review the code to see how how to set up key classes like `TVIVideoClient`, `TVIRoom`, `TVIParticipant`, `TVILocalMedia`, and `TVICameraCapturer`. The ViewController implements the `TVIRoomDelegate`, and `TVIParticipantDelegate` protocols in order to display remote Participant video on screen.
 
-### Additional Examples
+Download this project and run `pod install` to install TwilioVideo.framework. Open VideoQuickStart.xcworkspace in Xcode.
 
-You will also find additional examples that provide more advanced use cases of the Video SDK. The currently included examples are as follows:
 
-- [Screen Capturer](ScreenCapturerExample) - Shows how to use `TVIScreenCapturer` to capture the contents of a `UIView`, and how a custom `TVIVideoCapturer` can be implemented to do the same.
 
-## Prerequisites
+## Quickstart
 
-This project uses Apple's Swift programming language 3.0 for iOS, and the only supported way to develop iOS apps is on an Apple computer running OS X and Xcode. We have tested this application with the latest versions of iOS (10.0) and Xcode (8.0) at the time of this writing.
 
-You can use the iOS Simulator that comes with Xcode to do your testing, but local video will not be shared since the Simulator cannot access a camera. If you have an iOS device, you can now run apps from Xcode on your device without a paid developer account.
 
-## Twilio Library Setup for the Project
+### Running the Quickstart
 
-You will need to add the Twilio Video library to the project to compile and run. You can do this using the [Cocoapods](https://cocoapods.org/) dependency management system. Version 1.0 or greater is required.
+To get started with the Quickstart application follow these steps:
 
-### Using Cocoapods
+1.Open this project in xCode and select the quickstart app module
 
-First, you will need to have Cocoapods 1.0.0+ installed on your Mac, so go ahead and do that if you haven't already - the directions are here: [Getting Started with Cocoapods](https://guides.cocoapods.org/using/getting-started.html). If you're not sure, type `pod --version` into a command line.
+<img width="700px" src="images/quickstart/xcode-video-quickstart.png"/>
 
-Next, just run `pod install` from the command line in the top level directory of this project. Cocoapods will install the Twilio library and then set up a .xcworkspace file that you will use to run your project from now on.
+2.Type in the identity and click on "Generate Access Token" from the [testing tools page](https://www.twilio.com/user/account/video/dev-tools/testing-tools). Pick a name for your user identity (such as "Alice"). Leave this web page open, because you'll use it as the other side of the video chat.
```

Comment:
Missing steps here? Step one has us looking at the project in Xcode. Step 2 has us in the Twilio console setting an identity and generating an access token.